### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by lukas-hofstaetter** [enbility/eebus-go#79] [General]
Hi @DerAndereAndi ,
thank you so much for your work. I notices that the latest release is more than 10 months old and that there are many new commits on the `dev` branch. The latest `dev` is running fine for me. Could you please create a new tag/release for it?
Berst Regards
Lukas

**Comment by DerAndereAndi** [comment]
Hi @lukas-hofstaetter ,

I am still polishing, especially on the CEMd repository which may need adjustements in this, SHIP and SPINE. Shouldn't be taking too long, and then all 4 repos will get a 0.5 release tag.

Best,
Andreas

**Comment by DerAndereAndi** [comment]
Version 0.5 has been released.
